crates/re_ui/src/icons.rs

```diff
@@ -109,15 +109,20 @@ pub const CONTAINER: Icon = Icon::new("container", include_bytes!("../data/icons
 
 pub const WELCOME_SCREEN_CONFIGURE: Icon = Icon::new(
     "welcome_screen_configure",
-    include_bytes!("../data/images/welcome_screen_configure.png"),
+    include_bytes!("../data/images/onboarding-configure.jpg"),
```

Comment:
nit: would probably look better as a png (its mostly graphics)

Reply:
It was a jpg for a reason (the balloon there was actually an image), but the asset have been changed again anyways soo moot point (all are png and tiny now).